src/modules/customer/HappyUserComponents/WizardForm.js

```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react';
```

Comment:
I would recommend to design it so it should be a reusable component in /components/WizardForm and should dynamically render forms based on props passed.

For now only create a bare component there. You will think on it better when you come to the implementation.